cf/commands/domain/delete_domain_test.go

```diff
@@ -63,6 +63,28 @@ var _ = Describe("delete-domain command", func() {
 		})
 	})
 
+	Context("Checks whether the domain is owned or shared", func() {
```

Comment:
The BeforeEach() sets the domain to 'Shared: true', so this context should be something like, "when the domain is shared".